go.mod

```diff
@@ -1,4 +1,6 @@
-module github.com/dubbogo/hessian2
+module github.com/apache/dubbo-go-hessian2
```

Comment:
the module name is already fixed in master branch, there is a conflict, pls fix it